### ISSUE TITLE
Export more data from dokuwiki metadata

### ISSUE THREAD

**Issue by TheKeymaster** [findologic/dokuwiki-plugin-findologic-xml-export#23]
At the moment we get the data with

```
        return $metadata["description"]["abstract"];
```

It looks like we get not enough data here if the page is too long. We could use

```
$metadata["description"]["tableofcontents"];
```

according to the DokuWiki metadata https://www.dokuwiki.org/devel:metadata.
And format it of course, because its an array with some data keys.

**Comment by TheKeymaster** [comment]
It looks like it is not possible to get the entire data from the page with the *metadata*.
In my test environment I have created a page with > 500 characters. 

* `$metadata["description"]["abstract"];`
    * Returns only the first 200 characters (Description).
* `$metadata["description"]["tableofcontents"];`
    * Returns an array with all headings (Maybe parse all words to the Keywords in the export?).

If I have following page:

-----

```
==== Demopage ====

..... > 500 characters .....

Some important information
```

-----

I am not able to get `Some important information` without directly opening the `.txt` file where this data is saved. As you also can see on this Screenshot:

![selection_106](https://user-images.githubusercontent.com/20555112/31380425-a174b168-adb1-11e7-9227-9de7d75f0431.png)

@georgms how should we proceed with this?
I think we actually want this data but we are not able to, because its not stored at some place where we can call it. We need to directly open the `.txt` file if we want to get this data.

**Comment by georgms** [comment]
@TheKeymaster Somehow Dokuwiki is able to display the entire page when opening it in a browser. How does it do that?

**Comment by TheKeymaster** [comment]
@georgms After searching for years (at least it felt like it :smile:) I have found the function:

-----
```php
rawWiki($id);
```
-----
This function returns the page data as string. I will use this function.

**Comment by TheKeymaster** [comment]
Fixed with #26
